src/Microsoft.AspNet.Mvc.Core/Formatters/JsonOutputFormatter.cs

```diff
@@ -16,13 +17,23 @@ public class JsonOutputFormatter : OutputFormatter, IJsonOutputFormatter
         private JsonSerializerSettings _serializerSettings;
 
         public JsonOutputFormatter()
```

Comment:
> Is `InternalsVisibleTo` setup?

Yes but not to every test project that uses the parameterless constructor.  Don't worry about it.